### PR TITLE
MeetingSessionStatusCode comments updated

### PR DESCRIPTION
**Pull request by georgezy-amzn**
## ℹ️ Description
Update the comment for each MeetingSessionStatusCode

### Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
    - [ ] README update
    - [ ] CHANGELOG update
    - [ ] guides update
- [ ] This change requires a dependency update
    - [ ] Amazon Chime SDK Media
    - [ ] Other (update corresponding legal documents)

## 🧪 How Has This Been Tested?
*describe the tests that you ran to verify your changes, any relevant details for your test configuration*

### Additional Manual Test
- [ ] Pause and resume remote video
- [ ] Switch local camera
- [ ] Rotate screen back and forth

## 📱 Screenshots, if available
*provide screenshots/video record if there's a UI change in demo app*

#### Integration validation (required before release)

- [ ] Unit tests pass
- [ ] Build SDK against simulator with release options
- [ ] Build SDK against device with release options
- [ ] Build SDK against simulator with debug options
- [ ] Build SDK against device with debug options
- [ ] Test Demo against simulator with SDK (debug build) in workspace
- [ ] Test Demo against device with SDK (debug build) in workspace
- [ ] Test DemoObjC against simulator with SDK (debug build) in workspace
- [ ] Test DemoObjC against device with SDK (debug build) in workspace
- [ ] Test Demo against simulator with SDK.framework (release build)
- [ ] Test Demo against device with SDK.framework (release build)
- [ ] Test DemoObjC against simulator with SDK.framework (release build)
- [ ] Test DemoObjC against device with SDK.framework (release build)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by xuesichao — APPROVED** [review]

**Review by ziyiz-amzn — APPROVED** [review]

**Review by dingyishen-amazon — APPROVED** [review]

File: AmazonChimeSDK/AmazonChimeSDK/session/MeetingSessionStatusCode.swift [hunk 1]
```diff
@@ -12,46 +12,48 @@ import Foundation
...
 
-    /// The audio leg failed.
+    /// There was an internal server error related to audio. This may indicate some issue with the audio device, or an issue with the Amazon Chime SDK service itself.
     case audioDisconnected = 9
 
     /// Due to connection health a reconnect has been triggered.
     case connectionHealthReconnect = 10
 
-    /// Network is not good enough for VoIP.
+    /// Network is not good enough for VoIP, `AudioVideoObserver.audioSessionDidDrop()` will be triggered, and there will an automatic attempt of reconnecting.
```
Comment (hvsw) [inline AmazonChimeSDK/AmazonChimeSDK/session/MeetingSessionStatusCode.swift:21]:
```suggestion
    /// Network is not good enough for VoIP, `AudioVideoObserver.audioSessionDidDrop()` will be triggered, and there will be an automatic attempt of reconnecting.
```

File: AmazonChimeSDK/AmazonChimeSDK/session/MeetingSessionStatusCode.swift [hunk 2]
```diff
@@ -12,46 +12,48 @@ import Foundation
...
+    /// If the reconnecting is successful, `onAudioSessionStarted` will be called with value of reconnecting as true.
```
Comment (hvsw) [inline AmazonChimeSDK/AmazonChimeSDK/session/MeetingSessionStatusCode.swift:22]:
```suggestion
    /// If the reconnecting is successful, `audioSessionDidStart(reconnecting:)` will be called with value of `reconnecting` as `true`.
```

File: AmazonChimeSDK/AmazonChimeSDK/session/MeetingSessionStatusCode.swift [hunk 3]
```diff
@@ -12,46 +12,48 @@ import Foundation
...
     /// The attendee joined from another device.
     case audioJoinedFromAnotherDevice = 61
 
-    /// There was an internal server error with the audio leg.
+    /// There was an internal server error related to audio. This may indicate some issue with the audio device, or an issue with the Amazon Chime SDK service itself
     case audioInternalServerError = 62
 
-    /// Authentication was rejected. The client is not allowed on this call.
+    /// Authentication was rejected as the attendee information in MeetingSessionCredentials did not match that of an attendee created via chime::CreateAttendee.
+    /// This error may imply an issue with your credential providing service, the client will not allowed on this call.
```
Comment (hvsw) [inline AmazonChimeSDK/AmazonChimeSDK/session/MeetingSessionStatusCode.swift:35]:
```suggestion
    /// This error may imply an issue with your credential providing service, the client will not be allowed on this call.
```

**Review by hvsw — APPROVED** [review]
Nice to see these improvements to better express the intent of each error!

**Review by ziyiz-amzn — APPROVED** [review]

**Review by dingyishen-amazon — APPROVED** [review]

**Pull request merged**